trieste/acquisition/function/greedy_batch.py

```diff
@@ -351,3 +355,268 @@ def __call__(self, x: TensorType) -> TensorType:
         p = -5  # following experiments of :cite:`Alvi:2019`.
         penalization = ((pairwise_distances / (self._radius + self._scale)) ** p + 1) ** (1 / p)
         return tf.reduce_prod(penalization, axis=-1)
+
+
+class FantasizeAcquisitionFunction(GreedyAcquisitionFunctionBuilder[ProbabilisticModel]):
```

Comment:
we don't have `AcquisitionFunction` in the names of other acquisition functions, except for this one and LP, shall we remove those parts and make it consistent with other names?

Reply:
```suggestion
class FantasizeAcquisitionFunction(GreedyAcquisitionFunctionBuilder[FastUpdateModel]):
```